config/settings.yml

```diff
@@ -1121,6 +1122,10 @@
       :event_handler:
         :cpu_usage_threshold: 0.percent
         :nice_delta: 7
+        :queue_hostname: localhost
+        :queue_port: 61616
+        :queue_username: admin
+        :queue_password: smartvm
```

Comment:
I think these are fine here if you don't want to use ENV based settings (e.g. development), but I think the code that fetches these settings should use the ENV variable if it is set.

[Here](https://github.com/ManageIQ/manageiq/blob/409df9dcd4940549380238279ab35ea93a2b51cb/lib/miq_memcached.rb#L5) is how we do it for memcached.